synapse/lib/scrape.py

```diff
@@ -116,7 +163,7 @@ def cve_check(match: regex.Match):
 
 _regexes = collections.defaultdict(list)
 for (name, rule, opts) in scrape_types:
-    blob = (regex.compile(rule, regex.IGNORECASE), opts)
+    blob = (regex.compile(rule, regex.IGNORECASE | regex.VERBOSE), opts)
```

Comment:
the verbose flag may change how other regular expressions are interpreted.

We could pull in the `flags` option here https://github.com/vertexproject/synapse/pull/3232/files#diff-06ff0712dafce44871772e6b5df00a36703784789cce5fff7f76f6ed8b586121R131 and then embed flag information in the configuration dictionary for the entry?

Reply:
Great idea, thanks!